products/kirby.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Kirby
+category: server-app
+iconSlug: kirby
+permalink: /kirby
+alternate_urls: 
+-   /getkirby
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: true
+releaseLabel: "__RELEASE_CYCLE__{%if r.codename %} ({{r.codename}}){%endif%}"
+changelogTemplate: https://getkirby.com/releases/{{"__LATEST__"|split:'.'|slice:0,2|join:'.'}}
+
+auto:
+-   git: https://github.com/getkirby/kirby.git
+
+releases:
+
+-   releaseCycle: "3.9"
+    releaseDate: 2023-01-17
+    latest: "3.9.5"
+    latestReleaseDate: 2023-06-07
+    codename: "Brookesia"
+    eol: false
+
+-   releaseCycle: "3.8"
+    releaseDate: 2022-10-06
+    latest: "3.8.4"
+    latestReleaseDate: 2023-01-10
+    codename: "Rhampholeon"
+    eol: 2025-01-16
+
+-   releaseCycle: "3.7"
+    releaseDate: 2022-06-27
+    latest: "3.7.5.1"
+    latestReleaseDate: 2022-10-18
+    codename: "Kinyongia"
+    eol: 2024-10-05
+
+-   releaseCycle: "3.6"
+    releaseDate: 2021-11-16
+    latest: "3.6.6.2"
+    latestReleaseDate: 2022-10-18
+    codename: "Jungle Calumma"
+    eol: 2024-06-26
+
+-   releaseCycle: "3.5"
+    releaseDate: 2020-12-15
+    latest: "3.5.8.2"
+    latestReleaseDate: 2022-10-18
+    codename: "Calumma"
+    eol: 2023-11-15
+
+-   releaseCycle: "3.4"
+    releaseDate: 2020-07-07
+    latest: "3.4.5"
+    latestReleaseDate: 2020-12-01
+    link: https://github.com/getkirby/kirby/releases/tag/3.4.5
+    codename: "Furcifer"
+    eol: 2021-11-16
+
+-   releaseCycle: "3.3"
+    releaseDate: 2019-11-05
+    latest: "3.3.6"
+    latestReleaseDate: 2020-04-27
+    link: https://github.com/getkirby/kirby/releases/tag/3.3.6
+    codename: "Trioceros"
+    eol: 2021-11-16
+
+-   releaseCycle: "3.2"
+    releaseDate: 2019-06-25
+    latest: "3.2.5"
+    latestReleaseDate: 2019-09-24
+    link: https://github.com/getkirby/kirby/releases/tag/3.2.5
+    codename: "Archaius"
+    eol: 2021-11-16
+
+-   releaseCycle: "3.1"
+    releaseDate: 2019-03-19
+    latest: "3.1.4"
+    latestReleaseDate: 2019-05-21
+    link: https://github.com/getkirby/kirby/releases/tag/3.1.4
+    codename: "Chamaeleo"
+    eol: 2021-11-16
+
+-   releaseCycle: "3.0"
+    releaseDate: 2019-02-05
```

Comment:
https://packagist.org/packages/getkirby/cms#3.0.0 and https://github.com/getkirby/kirby/commit/72f01eac0a7db59360894e0cfb44680276260174 say 3.0.0 was tagged and published on 15th Jan. 

The GitHub release was created later.

```suggestion
    releaseDate: 2019-01-15
```